test/unit/views/disassociateucr/DisassociateUcrViewSpec.scala

```diff
@@ -64,7 +53,7 @@ class DisassociateUcrViewSpec extends ViewSpec with MockitoSugar with BeforeAndA
     }
 
     "display 'Back' button that links to Choice when ducrPart disabled" in {
-      when(appConfig.isDucrPartsEnabled).thenReturn(false)
+      when(config.isDucrPartsEnabled).thenReturn(false)
```

Comment:
The `afterEach` part was necessary. Still you want to reset the mock after every test case

Reply:
Are you sure?

Reply:
yeah, you should have defined the mock in every test case using method `beforeEach` if it's the same setting for everything or define it manually for every test case.

Every mock that is used should be reset in afterEach method, because reset should be always after every test case

Reply:
I know it's good practice and what we usually do.  But there is no real need to reset the mock every time.  Especially a simple mock of a boolean response.

So what would you like?   I originally was setting the mock behaviour in before.  Only 2 of the tests care what the mock returns - they set the mock manually.   The others work without setting the mock (as is currently the case).

Reply:
Approved, I prefer in general the approach with afterEach etc methods, but it's not an issue in that simple mock, current approach works